peps/pep-0728.rst

```diff
@@ -0,0 +1,614 @@
+PEP: 728
+Title: TypedDict with Typed Extra Fields
+Author: Zixuan James Li <zixuan.li@nyu.edu>
+Sponsor: Alex Waygood <alex.waygood@gmail.com>
+Discussions-To: https://discuss.python.org/t/pep-692-using-typeddict-for-more-precise-kwargs-typing/17314
```

Comment:
Should be a new thread, not the one for PEP 692

Reply:
Before this is merged, should I create a new "Typing" thread or leave it blank until a "pep" thread can be created?